tests/components/telegram_bot/test_telegram_bot.py

```diff
@@ -974,3 +974,39 @@ async def test_send_video(
     await hass.async_block_till_done()
     assert mock_get.call_count > 0
     assert response["chats"][0]["message_id"] == 12345
+
+
+async def test_set_message_reaction(
+    hass: HomeAssistant,
+    mock_broadcast_config_entry: MockConfigEntry,
+    mock_external_calls: None,
+) -> None:
+    """Test set message reaction."""
+    mock_broadcast_config_entry.add_to_hass(hass)
+    await hass.config_entries.async_setup(mock_broadcast_config_entry.entry_id)
+    await hass.async_block_till_done()
+
+    with patch(
+        "homeassistant.components.telegram_bot.bot.TelegramNotificationService.set_message_reaction",
```

Comment:
oh I missed this in my previous review. We should avoid patching internal integration methods and instead patch the dependency methods. 
In this case, can we patch `homeassistant.components.telegram_bot.bot.Bot.set_message_reaction`?

Reply:
Done. (I changed also 2 other tests that patched TelegramNotificationService)